src/objects.js

```diff
@@ -1,41 +1,59 @@
 const createPerson = (name, age) => {
-  // your code here
+return {
+  name: name,
+  age: age,
+
+};
 };
 
 const getName = object => {
-  // your code here
+return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+return object.hasOwnProperty;
 };
 
 const isOver65 = person => {
-  // your code here
+if (person.age > 65) {
+  return true;
+} else {
+  return false;
+}
 };
 
 const getAges = people => {
-  // your code here
+const arrayAges = people.map(person => person.age);
+return arrayAges;
 };
 
 const findByName = (name, people) => {
-  // your code here
+const person = people.find(person => person.name === name);
+return person;
 };
 
 const findHondas = cars => {
-  // your code here
+const hondas = cars.filter(car => car.manufacturer === 'Honda');
+return hondas;
 };
 
 const averageAge = people => {
-  // your code here
+let totalAge = 0;
+for (i=0; i < people.length; i++) {
+  totalAge += people[i].age;
+
+return totalAge/people.length;
+}
 };
 
 const createTalkingPerson = (name, age) => {
-  // your code here
+function introduce(name) {
+return 'Hi ${name}, my name is ${name} and I am ${age}!';
+}
 };
```

Comment:
In this test, you are supposed to return an object with the properties `name`, `age` and `introduce`, of which introduce is a `method` that takes one parameter (a different name) and returns a string. The final result ends up looking something like:
```
const createTalkingPerson = (name, age) => {
  return {
    name,
    age,
    introduce(person) {
      return `Hi ${person}, my name is ${this.name} and I am ${this.age}!`;
    }
  };
};

```